tower/Cargo.toml

```diff
@@ -25,6 +25,25 @@ edition = "2018"
 
 [features]
 default = ["log"]
+full = [
+  "log",
```

Comment:
the "log" feature is already turned on by default, so I don't think it should be part of this feature. That way, if users want to enable all the _middleware_ without using the `log` crate, they can write
```rust
tower = { version = "0.4", default_features = false, features = ["full"] }
```

Reply:
Makes sense 👍 Fixed in bf1c665